docker-build-system/zombodb-build-alpine-3.12/Dockerfile

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright 2018-2020 ZomboDB, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+FROM rust:1.47-alpine3.12 as zombodb-extension
```

Comment:
this is inconsistent with the other Dockerfiles here.  I want to `FROM <base distro>:<version>` and that's it.

Reply:
it’s fixed